pkgs/tools/system/clinfo/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl, ocl-icd, opencl-headers }:
+
+stdenv.mkDerivation rec {
+  pname = "clinfo";
+  version = "2.2.18.04.06";
+
+  src = fetchurl {
+    url = "https://github.com/Oblomov/${pname}/archive/${version}.tar.gz";
+    sha256 = "f77021a57b3afcdebc73107e2254b95780026a9df9aa4f8db6aff11c03f0ec6c";
+  };
+
+  buildInputs = [ ocl-icd opencl-headers ];
+
+  NIX_LDFLAGS = [ "-lOpenCL" ];
```

Comment:
Why is this necessary? They appear to already be setting it https://github.com/Oblomov/clinfo/blob/9d3f66f603d1e76173ed42be1d9181c991fb4226/Makefile#L43